client/src/components/availability/SelectTimeForm.js

```diff
@@ -81,16 +81,16 @@ function AddTimeForm(props){
             start: new Date(year,month,date,start_hour,start_minute),
             end: new Date(year,month,date,end_hour,end_minute)
         }
-        setAvailability(availability.push(newDate))
+        setAvailability(availability.push(newDate));
```

Comment:
Should this be `setAvailability([...availability, newDate]);` like your Slack message? Right now you're mutating state directly which is not a good practice in React